src/main/java/org/openRealmOfStars/AI/Mission/MissionList.java

```diff
@@ -88,7 +90,9 @@ public Mission getMissionForFleet(final String fleetName,
   public Mission getMissionForFleet(final String fleetName) {
     for (Mission mission : missions) {
       if (mission.getFleetName() != null
-          && mission.getFleetName().equals(fleetName)) { return mission; }
```

Comment:
I noticed that this is what Eclipse did after tuning the clean up settings. I tried to get rid of these but I haven't find the correct setting to for if clauses with newlines.